### PR TITLE
add project progression doc

### PR DESCRIPTION
**Pull request by obensource**
Copied verbatim from /bootstrap.

Let me know if commit history preservation is an issue. 👍

### CODE REVIEW

File: PROJECT_PROGRESSION.md [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+* Projects get accepted via a 2/3 supermajority vote of the CPC.
+* The proposal document will be finalized as a project charter. This charter document must be included in the project's main repository.
+* The CPC will determine the appropriate initial stage for the project. The project can apply for a different stage via the review process. 
+
+## III. Stages - Definitions & Expectations
+Every Foundation project has an associated maturity level. Proposed Foundation projects should state their preferred maturity level. Projects of all maturities have access to Foundation resources.
+
+All Foundation projects may attend CPC meetings and contribute work regardless of their stage. 
+
+*note: all stage names are tbd pending outcome of [#44](https://github.com/nodejs/bootstrap/issues/44#issuecomment-440026298)*
```
Comment (amorist) [inline PROJECT_PROGRESSION.md]:
```suggestion
*note: all stage names are tbd pending outcome of [#44](https://github.com/openjs-foundation/bootstrap/issues/44#issuecomment-440026298)*
```

**Comment by obensource** [comment]
Thanks a million @amorist! 🙌

**Comment by MylesBorins** [comment]
This doc should likely stay in the CPC repo. Changes to the doc should also happen through meetings and not through this PR. Lets chat in today's meeting to figure out what is the best way forward on this

**Comment by obensource** [comment]
Given https://github.com/openjs-foundation/admin/issues/14, I'm closing this PR. 👍

**Pull request closed without merging**